lib/jmap/core/method/method.dart

```diff
@@ -4,6 +4,8 @@ import 'package:jmap_dart_client/jmap/core/capability/capability_identifier.dart
 import 'package:jmap_dart_client/jmap/core/request/request_invocation.dart';
 
 abstract class Method with EquatableMixin {
+  const Method();
```

Comment:
why we need it?

Reply:
At first it was to make calendar event reply method `const`, but due to the nature of `Optional` mixin, `const` constructor cannot be achieved. Reverted.